.travis.yml

```diff
@@ -17,7 +22,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
 
 script:
-  - BASEURL=http://127.0.0.1:9500 make all
+  - BASEURL=http://127.0.0.1:9500 PLATFORM=$PLATFORM BROWSER=$BROWSER make all
```

Comment:
Does this mean that the build will take longer, or is the matrix only for sauce tests?